### DISCUSSION TITLE
Token for direct API call

### DISCUSSION THREAD

**Discussion by mjdNjhNJ** [pocketbase/pocketbase#4785] [Q&A] [answered]
I'm trying to directly call API. In PB admin board it says: "Requires admin Authorization:TOKEN header". Where do I generate such token?
```
    const xhr = new XMLHttpRequest();
    xhr.open("GET", "http://127.0.0.1:8090/api/collections/players/records");
    xhr.setRequestHeader("Authorization", "TOKEN");
    xhr.send();
    xhr.responseType = "json";
    xhr.onload = () => {
            if (xhr.readyState == 4 && xhr.status == 200) {
                const data = xhr.response;
            } else {
                console.log(`Error: ${xhr.status}`);
            }
        };
```

**Comment by ganigeorgiev** [comment, marked as answer]
For generating an admin token you need to send admin auth with password request:
- https://pocketbase.io/docs/api-admins/#auth-with-password

For the regular users, see:
- https://pocketbase.io/docs/api-records/#auth-with-password
- https://pocketbase.io/docs/api-records/#auth-with-oauth2

If the above doesn't help, please don't just "prompt" and try elaborate a little more on your use case to understand the broader picture and why using the [JS SDK](https://github.com/pocketbase/js-sdk) or even [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) is not an option.
